<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sort</class>
 <widget class="QDialog" name="Sort">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>162</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>162</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>2000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="Distant_rbtn">
         <property name="text">
          <string>Дальность</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Distant_box">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>Нет</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>До 100</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>До 1000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>До 10000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>До 10000+</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="Name_rbtn">
         <property name="text">
          <string>Имя</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Name_line">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>295</width>
           <height>0</height>
          </size>
         </property>
         <property name="maxLength">
          <number>1</number>
         </property>
         <property name="placeholderText">
          <string>Введите первый символ...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QRadioButton" name="Type_rbtn">
         <property name="text">
          <string>Тип</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Type_line">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>295</width>
           <height>0</height>
          </size>
         </property>
         <property name="maxLength">
          <number>1</number>
         </property>
         <property name="placeholderText">
          <string>Введите первый символ...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QRadioButton" name="Time_rbtn">
       <property name="text">
        <string>Время создания</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OK_btn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>ОК</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel_btn">
       <property name="text">
        <string>Выход</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
